css/common.css

```diff
@@ -1,75 +1,80 @@
-.footer{
-	width: 100%;
-	
-	line-height: 100px;
-	text-align: center;
-	color: #4d4d4d;
-	background-color: #bababa;	
+.footer {
+    width: 100%;
+    line-height: 100px;
+    text-align: center;
 }
 
-.logo{
-	width: 200px;
-	height: 37.40px;
-	margin-top: 20px;
-	margin-left: 50px;
-	margin-right: 150px;
-
-	float: left;
-	background-image: url('https://www.softvision.ro/images/common/logo-v2.png');
-	background-size: contain;
+.footerSkin {
```

Comment:
in general in CSS the convention is not camel case but footer-skin('-' as a separator)